src/index.tsx

```diff
@@ -61,6 +61,7 @@ const Terminal = ({name, prompt, height = "600px", colorMode, onInput, children,
       onInput(currentLineInput);
       setCursorPos(0);
       setCurrentLineInput('');
+      scrollIntoViewRef?.current?.scrollIntoView({ behavior: "auto", block: "center" });
```

Comment:
Merged this, but did a little additional testing (in Firefox) after merge. This doesn't seem to be working quite right:
- Entire terminal now moves down on first Enter; the terminal should not change positions
- When reaching the bottom, the cursor now is hidden slightly
Is there a reason you decided to change it from `block: nearest` to `block: center`?

Reply:
Hey Jon, hope you're doing great!
I changed from nearest to center because it also fixed some scrollIntoView bugs, I have some loom videos to show if you'd like to see... I'll attach it

[using-nearest.webm](https://github.com/jonmbake/react-terminal-ui/assets/80011318/ce13cfca-a159-42ae-a9b7-7469aaedc980)

[using-center.webm](https://github.com/jonmbake/react-terminal-ui/assets/80011318/f6340a59-e74d-4d40-bd4e-4a31e16fb2ea)

Reply:
The ScrollIntoView is intended to get the cursor to scroll into view if the inputs go longer than the terminal height-- not to try to scroll back into view if the user manually scrolls up or down on the page.

This change introduces a new bug where the cursor is hidden when it reaches the bottom.